st2common/bin/st2ctl

```diff
@@ -173,10 +173,10 @@ function clean_logs() {
 
 function getpids() {
   echo "##### st2 components status #####"
-  COMPONENTS=${COMPONENTS/mistral/mistral-server mistral-api}
+  COMPONENTS=${COMPONENTS/mistral/mistral-server mistral.api}
 
   for COM in ${COMPONENTS}; do
-    PID=`ps ax | grep -v grep | grep -v st2ctl | grep "${COM}" | awk '{print $1}'`
+    PID=`ps ax | grep -v grep | grep -v st2ctl | grep -E "(${COM}\.wsgi)|(bin/${COM})|(hubot ${COM})" | awk '{print $1}'`
```

Comment:
I think that's fine since we don't officially support FreeBSD / OS X or similar, because, IIRC, `-E` only works with GNU grep and there are some other differences with BSD grep.

Reply:
Looks like macOS FreeBSD version of grep does support `-E`. It does my head in dealing with variations of `grep`, `sed`, and `awk`.

But yeah, shouldn't matter here since we're only officially supporting GNU/Linux systems.